libraries/chain/include/eosio/chain/fork_database.hpp

```diff
@@ -136,8 +136,12 @@ namespace eosio::chain {
     * All methods assert until open() is closed.
     */
    class fork_database {
+      static constexpr uint32_t magic_number = 0x30510FDB;
```

Comment:
Only one magic number for both?

Reply:
yes, different versions. There is a single file.